### PR TITLE
Simplify FSS implementation

### PR DESCRIPTION
**Pull request by LaRiffle**
## Description
Some cleaning was need to make this code easier to read!

### CODE REVIEW

File: src/sympc/protocol/fss/fss.py [hunk 1]
```diff
@@ -133,20 +51,21 @@ def mask_builder(
...
+        ShareTensor: share of the masked input
     """
     x = x1 - x2
 
     keys = session.crypto_store.get_primitives_from_store(
         f"fss_{op}", nr_instances=x.numel(), remove=False
     )
 
-    alpha = np.frombuffer(np.ascontiguousarray(keys[:, 0:N]), dtype=np.uint32)
+    n_bytes = int(n / 8)  # keys contains bytes not bits
```
Comment (gmuraru) [inline src/sympc/protocol/fss/fss.py:67]: NITPICK: `n // 8`

File: src/sympc/protocol/fss/fss.py [hunk 2]
```diff
@@ -232,99 +110,81 @@ def comp_evaluate(session: Session, b, x_masked, dtype=None) -> ShareTensor:
...
 
-        Returns:
-            # noqa: DAR201
-        """
-        # x = x.astype(np.uint64)
-        original_shape = x.shape
-        x = x.reshape(-1)
-        flat_result = dif.eval(b, x, k_b)
-        return flat_result.astype(np.int32).astype(np.int64).reshape(original_shape)
+    response = MPCTensor(session=session, shares=shares)
```
Comment (gmuraru) [inline src/sympc/protocol/fss/fss.py:160]: NITPICK: You could add the `shape` in the `MPCTensor` (send it as an argument to the constructor)

**Review by gmuraru — APPROVED** [review]
LGTM!

File: src/sympc/protocol/fss/fss.py (hunk 2 above)
Comment (LaRiffle) [reply src/sympc/protocol/fss/fss.py]: Done :)

**Pull request merged**